moi.gov.sa,"HTTPSConnectionPool(host='moi.gov.sa', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fece9ee9a00>, 'Connection to moi.gov.sa timed out. (connect timeout=2)'))","HTTPConnectionPool(host='moi.gov.sa', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fece9f05130>, 'Connection to moi.gov.sa timed out. (connect timeout=2)'))"
moit.gov.vn,"HTTPSConnectionPool(host='moit.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='moit.gov.vn', port=80): Read timed out. (read timeout=2)"
moj.go.jp,"HTTPSConnectionPool(host='moj.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece9ee98e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moj.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece99d80a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moj.gov.sa,"HTTPSConnectionPool(host='moj.gov.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moj.gov.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
moja-delatnost.rs,"HTTPSConnectionPool(host='moja-delatnost.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='moja-delatnost.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mojakompanija.com,"HTTPSConnectionPool(host='www.mojakompanija.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mojakompanija.com', port=443): Read timed out. (read timeout=2)"
mojaljekarna.hr,"HTTPSConnectionPool(host='mojaljekarna.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mojaljekarna.hr', port=443): Read timed out. (read timeout=2)"
mojecelebrity.cz,"HTTPSConnectionPool(host='mojecelebrity.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mojecelebrity.cz' doesn't match 'partner.tiscali.cz'"")))","HTTPSConnectionPool(host='www.mojecelebrity.cz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece851a1c0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
mojekarte.si,"HTTPSConnectionPool(host='www.mojekarte.si', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.mojekarte.si', port=80): Read timed out. (read timeout=2)"
mommyasia.id,"HTTPSConnectionPool(host='mommyasia.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mommyasia.id', port=80): Read timed out. (read timeout=2)"
momoshop.com.tw,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mondaq.com,"HTTPSConnectionPool(host='mondaq.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece95b1100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mondaq.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece76f82b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mondovolantino.it,"HTTPSConnectionPool(host='mondovolantino.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mondovolantino.it' doesn't match 'www.mondovolantino.it'"")))","HTTPSConnectionPool(host='mondovolantino.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mondovolantino.it' doesn't match 'www.mondovolantino.it'"")))"
moneyandbanking.co.th,"HTTPSConnectionPool(host='moneyandbanking.co.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moneyandbanking.co.th', port=443): Read timed out. (read timeout=2)"
montreal.qc.ca,"HTTPSConnectionPool(host='montreal.qc.ca', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece851a280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='montreal.qc.ca', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece4f49c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
moov.ooo,"HTTPSConnectionPool(host='moov.ooo', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='moov.ooo', port=443): Read timed out. (read timeout=2)"
moph.go.th,"HTTPSConnectionPool(host='moph.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece978d160>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='moph.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece4a0f670>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mopt.go.cr,"HTTPSConnectionPool(host='mopt.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece7d56160>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mopt.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece4612370>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
morelos.gob.mx,"HTTPSConnectionPool(host='morelos.gob.mx', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='morelos.gob.mx', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
morgenbladet.no,"HTTPSConnectionPool(host='www.morgenbladet.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.morgenbladet.no', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece9d2b1c0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
mosquitoproject.org,"HTTPSConnectionPool(host='mosquitoproject.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mosquitoproject.org', port=80): Read timed out. (read timeout=2)"
motorguia.net,"HTTPSConnectionPool(host='motorguia.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='motorguia.net', port=443): Read timed out. (read timeout=2)"
mots.go.th,"HTTPSConnectionPool(host='mots.go.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mots.go.th', port=443): Read timed out. (read timeout=2)"
movistar.co,"HTTPSConnectionPool(host='www.movistar.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.movistar.co', port=443): Read timed out. (read timeout=2)"
movistarplus.es,"HTTPSConnectionPool(host='movistarplus.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'movistarplus.es' doesn't match either of '*.dof6.com', 'dof6.com'"")))","HTTPSConnectionPool(host='www.movistar.es', port=443): Read timed out. (read timeout=2)"
mpsp.mp.br,"HTTPSConnectionPool(host='mpsp.mp.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece595bbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mpsp.mp.br', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece162ff40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mrdiy.com,"HTTPSConnectionPool(host='www.mrdiy.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mrdiy.com', port=443): Read timed out. (read timeout=2)"
mrdiy.com.my,"HTTPSConnectionPool(host='www.mrdiy.com.my', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mrdiy.com.my', port=443): Read timed out. (read timeout=2)"
mscbs.gob.es,"HTTPSConnectionPool(host='mscbs.gob.es', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fecdf5fc0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mscbs.gob.es', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece9bc5820>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
msp.gob.ec,"HTTPSConnectionPool(host='msp.gob.ec', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fecdf5b9610>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='msp.gob.ec', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece8944400>: Failed to establish a new connection: [Errno 111] Connection refused'))"
mspas.gob.gt,"HTTPSConnectionPool(host='mspas.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='mspas.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
mssader.com,"HTTPSConnectionPool(host='mssader.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mssader.com', port=443): Read timed out. (read timeout=2)"
msudrf.ru,"HTTPSConnectionPool(host='msudrf.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='msudrf.ru', port=80): Read timed out. (read timeout=2)"
mt.gov.vn,"HTTPSConnectionPool(host='mt.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mt.gov.vn', port=443): Read timed out. (read timeout=2)"
mtps.gob.sv,"HTTPSConnectionPool(host='mtps.gob.sv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fece9946850>, 'Connection to mtps.gob.sv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='mtps.gob.sv', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fece0519910>, 'Connection to mtps.gob.sv timed out. (connect timeout=2)'))"
mtroos.com,"HTTPSConnectionPool(host='mtroos.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mtroos.com', port=443): Read timed out. (read timeout=2)"
mts.ru,"HTTPSConnectionPool(host='moskva.mts.ru', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mts.ru', port=443): Read timed out. (read timeout=2)"
mufg.jp,"HTTPSConnectionPool(host='mufg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece9aa5cd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mufg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf3fe4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
multicoisas.com.br,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mummum.dk,"HTTPSConnectionPool(host='mummum.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mummum.dk', port=443): Read timed out. (read timeout=2)"
mumsnet.com,"HTTPSConnectionPool(host='www.mumsnet.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mumsnet.com', port=443): Read timed out. (read timeout=2)"
municipiosefreguesias.pt,"HTTPSConnectionPool(host='municipiosefreguesias.pt', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='municipiosefreguesias.pt', port=80): Read timed out. (read timeout=2)"
murciasalud.es,"HTTPSConnectionPool(host='murciasalud.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'murciasalud.es' doesn't match 'www.murciasalud.es'"")))","HTTPConnectionPool(host='www.murciasalud.es', port=80): Read timed out. (read timeout=2)"
museumthailand.com,"HTTPSConnectionPool(host='museumthailand.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fecdf5fc970>, 'Connection to museumthailand.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='museumthailand.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fece9981850>, 'Connection to museumthailand.com timed out. (connect timeout=2)'))"
musicatm.com,"HTTPSConnectionPool(host='www.musicatm.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.musicatm.com', port=443): Read timed out. (read timeout=2)"
musor.tv,Exceeded 30 redirects.,Exceeded 30 redirects.
muzika.hr,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
muzikbahcesi.com,"HTTPSConnectionPool(host='muzikbahcesi.com', port=443): Read timed out.","HTTPSConnectionPool(host='muzikbahcesi.com', port=443): Read timed out."
mvb.lt,"HTTPSConnectionPool(host='mvb.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece4647640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mvb.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf251970>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mvep.hr,"HTTPSConnectionPool(host='mvep.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece7bc0760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mvep.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece89bd4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
mydns.jp,"HTTPSConnectionPool(host='mydns.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece8785850>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='mydns.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdfcad2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
myntra.com,"HTTPSConnectionPool(host='www.myntra.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.myntra.com', port=443): Read timed out. (read timeout=2)"
myq-see.com,"HTTPSConnectionPool(host='myq-see.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece75cda60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='myq-see.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf14f550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
myshop.lk,"HTTPSConnectionPool(host='www.myshop.lk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.myshop.lk', port=443): Read timed out. (read timeout=2)"
mysound.jp,"HTTPSConnectionPool(host='mysound.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mysound.jp', port=443): Read timed out. (read timeout=2)"
n-sport.net,"HTTPSConnectionPool(host='n-sport.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece5f71730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='n-sport.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf11fee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
n11.com,"HTTPSConnectionPool(host='n11.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece21baf70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='n11.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecea1fa1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
na.se,"HTTPSConnectionPool(host='na.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.na.se', port=443): Read timed out. (read timeout=2)"
nabu.de,"HTTPSConnectionPool(host='nabu.de', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nabu.de', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
naftemporiki.gr,"HTTPSConnectionPool(host='naftemporiki.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='naftemporiki.gr', port=443): Read timed out. (read timeout=2)"
nagano.lg.jp,"HTTPSConnectionPool(host='nagano.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece73965b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nagano.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf4b3e50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nagoya-u.ac.jp,"HTTPSConnectionPool(host='nagoya-u.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece8dd5370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nagoya-u.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece95e0370>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nairobiwire.com,"HTTPSConnectionPool(host='nairobiwire.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nairobiwire.com', port=80): Read timed out. (read timeout=2)"
najlacnejsie-knihy.sk,"HTTPSConnectionPool(host='najlacnejsie-knihy.sk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='najlacnejsie-knihy.sk', port=443): Read timed out. (read timeout=2)"
nam.fi,"HTTPSConnectionPool(host='nam.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fecea8da280>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nam.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece8282fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
naoussa.be,"HTTPSConnectionPool(host='naoussa.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece80a3d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='naoussa.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fecdf7c6b80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nasdaq.com,"HTTPSConnectionPool(host='www.nasdaq.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.nasdaq.com', port=80): Read timed out. (read timeout=2)"
nasezdravie.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nationalarchives.gov.uk,"HTTPSConnectionPool(host='nationalarchives.gov.uk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='nationalarchives.gov.uk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
natmat.se,"HTTPSConnectionPool(host='natmat.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='natmat.se', port=443): Read timed out. (read timeout=2)"
natural-soap-directory.org,"HTTPSConnectionPool(host='natural-soap-directory.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fece84dd6d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='natural-soap-directory.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fece7a3bca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
